README.md

```diff
@@ -396,6 +430,65 @@ NOTE: For more SDK configuration options, please [see our SDK's configuration do
 
 NOTE: For content schema, please [use the example en-US locale file](/src/locales/us/en/index.ts).
 
+### Configuration
+
+This object can be useful when you want to leverage APIs that require interaction with the ForgeRock platform or access stored tokens in a location of your app that may not directly use the `Widget`. When the `configuration.set` method is used, it removes the need to set the `config` property within the `Widget` class instantiation.
+
+For example, these would be equivalent:
+
+```js
+// Configuration within Widget instantiation
+const widget = new Widget({
+  target: document.getElementById('widget-root'),
+  props: {
+    config: {
+      /**
+       * REQUIRED; SDK configuration object
+       */
+      serverConfig: {
+        baseUrl: 'https://customer.forgeblocks.com/am',
+      },
```

Comment:
Just something to note, I found that it is mandatory to set a timeout in the `serverConfig`. The examples here imply that it is optional, however an error is thrown if you follow this:

```
ERROR Error: Uncaught (in promise): [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "undefined",
    "path": [
      "serverConfig",
      "timeout"
    ],
    "message": "Required"
  }
]
```

I can see in [sdk.config.ts](https://github.com/ForgeRock/forgerock-web-login-framework/blob/85de439122efc9b80cf432846fba716c3037631e/src/lib/sdk.config.ts#L52) that there is a comment acknowledging that it should be optional. Given this, I have two questions:

1) Do we have a ticket open for this on the JS SDK? I couldn't see one but maybe I was searching for the wrong thing.
2) Should we make this mandatory in the docs until it is fixed in the JS SDK?

Reply:
I think this is part of the SDK type fixes that we have a story for addressing. @ryanbas21 can you double check this?